### PR TITLE
V2.1.2

### PR DESCRIPTION
**Pull request by mdeceglie**
- [x] Code changes are covered by tests
- [x] Code changes have been evaluated for compatibility/integration with TrendAnalysis
- [x] New functions added to `__init__.py`
- [x] API.rst is up to date, along with other sphinx docs pages
- [x] Example notebooks are rerun and differences in results scrutinized
- [x] Updated changelog

Incorporates changes from #304 and #300

### CODE REVIEW

**Comment by mdeceglie** [comment]
To do: change the release date once this advances

File: docs/sphinx/source/changelog/v2.1.2.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+-------------
+* Preferred citation updated (:issue:`296`, :pull:`300`)
+
+Bug fixes
+---------
+* XGBoost model saved in a version-stable json. Removes the xgboost version restriction of version 2.1.1. (:issue:`301`, :pull:`304`)
+
+Requirements
+------------
+* ``joblib`` removed as requirement (:pull:`304`)
```
Comment (kanderso-nrel) [inline docs/sphinx/source/changelog/v2.1.2.rst:15]: In the past we've noted changes to `requirements.txt` here, may be good to do the same for `xgboost`

File: docs/sphinx/source/changelog/v2.1.2.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
+**************************
+v2.1.2 (December 20, 2021)
+**************************
+
+Documentation
```
Comment (kanderso-nrel) [inline docs/sphinx/source/changelog/v2.1.2.rst:5]: nitpick: thinking about the ordering of these sections, I think I'd move "Documentation" down to the bottom and have "bug fixes" at the top

**Review by kanderso-nrel — APPROVED** [review]
LGTM pending updating the release date

**Comment by kanderso-nrel** [comment]
Looks like the TrendAnalysis notebook lost its cell 2, is that intentional?

**Comment by mdeceglie** [comment]
> Looks like the TrendAnalysis notebook lost its cell 2, is that intentional?

Nope! Good catch

**Pull request merged**